### DISCUSSION TITLE
Support for Midi CC Messages - MSB, LSB, Program

### DISCUSSION THREAD

**Discussion by epfaller** [helgoboss/helgobox#1332] [Q&A] [answered]
Hi @all,
first, good job in developing ReaLern, it is a very powerful tool, great job.

I intend to setup reaper for live performances, VSTs and a hardware synth. I have several tabs, each of them stands for on song. The songs are separted in Verse, Chorus, etc. What I now need ist, when I switch (triggered by my midi key) to the next tab, send out CC messages with defined MSB, LSB and program change.
As far as I see there are only program changes possible, or am I too stupid to see the related configurations.

Is there a way to get it work in the described way? 

Thanks in advance.
Eric

**Comment by helgoboss** [comment, marked as answer]
Hmm, how do you send the MIDI messages? I wouldn't know how to send them whenever the project tab changes.

**Comment by epfaller** [comment]
Hi,
thanks for your quick response. I spent much time to get my problem work.
But I think, this is an repaer issue, in doing internal workflow.
According to my knowledge, which does not go into deep, I follow up 2 approaches:

1. CC with ReaControl, which works when loading the project, but not when switching to a tab.
2. MIDI message with the program change, but for this I have to start the project with the tab switch, which is unpleasant but works.

My naive thinking was that RealLern could do that for me.
Anyway, good work in developing this plugin, I will use ReaLern in my projects for the functions for which it was designed.

Greetings from bavaria and enjoy!
